lib/compile.py

```diff
@@ -25,7 +25,7 @@ def compile_script(script_file):
     keys_to_ignore = ["template"]
 
     for line in [line.strip() for line in definition]:
-        if line and ':' in line and not line[0] == '#':
+        if line != "" and ':' in line and not line[0] == '#':
```

Comment:
Is there a reason for making this change? I thought `if line` was generally preferred for this sort of thing.